src/LaminasModuleProvider.php

```diff
@@ -179,7 +181,7 @@ public function getHydratorConfig() : array
         return $this->convert($this->module->getHydratorConfig());
     }
 
-    public function getInputFilterConfig() /* : array */
+    public function getInputFilterConfig(): array
```

Comment:
This can't be changed: it's a BC break for downstream consumers that `extend` this class